### PR TITLE
fix: system status gives empty output if meshery is not running 

### LINKED ISSUE #1788: [mesheryctl] Check for latest release available not working when Meshery server is unavailable
**Description**
When Meshery server is available, mesheryctl version will perform the latest release check:
```
➜  ~ mesheryctl version
Client Version: v0.4.15 	  GitSHA: 51ca056186a824ad1c2b1bf979b95bdc46978d34
Server Version: v0.4.17 	  GitSHA: 06abac7
Checking for latest version of Meshery....
v0.4.17 is the latest Meshery release.
```

`mesheryctl version` no longer checks for latest release available when Meshery server is unavailable.

```
➜  ~ mesheryctl version
Client Version: v0.4.15 	  GitSHA: 51ca056186a824ad1c2b1bf979b95bdc46978d34
Server Version: Unavailable 	  GitSHA: Unavailable
Error Occurred: Get http://localhost:9081/api/server/version: dial tcp [::1]:9081: connect: connection refused
Could not communicate with Meshery at http://localhost:9081/api/server/version
Ensure that Meshery is available.
 See Meshery Documentation (https://docs.meshery.io) for help.
```

**Expected Behavior**
`mesheryctl version` should check for latest release available even when Meshery server is unavailable.

### LINKED ISSUE #1795: Mesheryctl system status gives empty output if meshery is not running
**Description**

Mesheryctl system status gives empty output if meshery is not running
```sh
./mesheryctl system status
Meshery status...
Name   Command   State   Ports
```
**Expected Behavior**

It should give some descriptive response if meshery is not running

### PR DESCRIPTION
**Pull request by kushthedude**
**Description**

This PR fixes #1795

### CODE REVIEW

File: mesheryctl/internal/cli/root/system/status.go [hunk 1]
```diff
@@ -33,15 +33,22 @@ var statusCmd = &cobra.Command{
...
 
 		start := exec.Command("docker-compose", "-f", utils.DockerComposeFile, "ps")
-		start.Stdout = os.Stdout
-		start.Stderr = os.Stderr
 
-		if err := start.Run(); err != nil {
-			return errors.Wrap(err, utils.SystemError("failed to start meshery"))
+		outputStd, err := start.Output()
+		if err != nil {
+			return errors.Wrap(err, utils.SystemError("failed to get meshery status"))
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/status.go:42]: Capitalize Meshery, if you would.

File: mesheryctl/internal/cli/root/system/status.go [hunk 2]
```diff
@@ -33,15 +33,22 @@ var statusCmd = &cobra.Command{
...
 		}
+
+		outputString := string(outputStd)
+		if strings.Contains(outputString, "meshery") {
+			log.Info(outputString)
+		} else {
+			log.Info("Meshery is not running, run `mesheryctl system start` to start meshery")
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/system/status.go:49]: Capitalize Meshery

File: mesheryctl/internal/cli/root/system/status.go (hunk 1 above)
Comment (kushthedude) [reply mesheryctl/internal/cli/root/system/status.go]: Done!

File: mesheryctl/internal/cli/root/system/status.go (hunk 2 above)
Comment (kushthedude) [reply mesheryctl/internal/cli/root/system/status.go]: Done!

**Review by leecalcote — APPROVED** [review]

**Pull request merged**